dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -688,8 +689,16 @@ public static int call(Logger logger, FuseCallable callable, String methodName,
   public static String getMountedRootPath(AlluxioConfiguration conf, FuseOptions fuseOptions) {
     Optional<UfsFileSystemOptions> options
         = fuseOptions.getFileSystemOptions().getUfsFileSystemOptions();
-    return options.isPresent() ? options.get().getUfsAddress()
-        : conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH);
+    String fuseMountAlluxioPath = conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH);
+    // If we have set an FUSE_MOUNT_ALLUXIO_PATH, use it as the "MountedRootPath".
+    if (!fuseMountAlluxioPath.isEmpty()
+        && (!fuseMountAlluxioPath.equals(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH.getDefaultValue()))) {
+      return fuseMountAlluxioPath;
+    }
+    if (options.isPresent())  {
+      return options.get().getUfsAddress();
+    }
+    return Strings.EMPTY;
```

Comment:
The old code is that if `getUfsFileSystemOptions()` is successful, then return the Ufs root address as the "MountedRootPath", while we have set the `PropertyKey.FUSE_MOUNT_ALLUXIO_PATH` where we expect alluxio to mount to. So the "MountedRootPath" is not the `FUSE_MOUNT_ALLUXIO_PATH` we expected, then asserts error.

Reply:
The values in ` Optional<UfsFileSystemOptions> options` have higher priority in my mind

Reply:
You are right, but if i change the priority, all the ignored tests(except 1) in AlluxioJniFuseFileSystemTest.java are passed. I'm confused and not sure of the correctness of my fix.